addons/hr_holidays/static/src/scss/accrual_plan_level.scss

```
@@ -1,17 +1,10 @@
 $o-hr-holidays-border-color: map-get($grays, '300');
 
 .o_hr_holidays_hierarchy {
-    margin-left: -$o-horizontal-padding;
-    margin-right: -$o-horizontal-padding;
-    @include media-breakpoint-up(lg, $o-extra-grid-breakpoints) {
-        margin-left: -$o-horizontal-padding*2;
-        margin-right: -$o-horizontal-padding*2;
-    }
-    margin-bottom:-24px;
-    padding: 10px 0px 24px 16px;
-    background-color: rgba(128, 128, 128, 0.15);
-    box-shadow: 0px 1px 1px rgba(17, 17, 17, 0.23);
-    overflow-x: auto;
+    margin-left:calc(var(--formView-sheet-padding-x) * -1);
+    margin-right: calc(var(--formView-sheet-padding-x) * -1);
+    margin-bottom: calc(var(--formView-sheet-padding-x) * -1);
+    padding: var(--formView-sheet-padding-x);
```

Missing empty row between selectors ;) 
```suggestion
    padding: var(--formView-sheet-padding-x);
    
```